test/test_flop_counter.py

```diff
@@ -675,6 +676,55 @@ def split_tensor(x):
             ),
         )
 
+    @skipIfRocm  # Nested tensor
+    @unittest.skipIf(not HAS_CUDA, "CUDA not available")
+    @unittest.skipIf(
+        not PLATFORM_SUPPORTS_FLASH_ATTENTION,
+        "Does not support all SDPA backends (pre-SM80 hardware on CUDA)",
+    )
+    def test_nested_attention_fake_tensors(self):
+        # shape_env = ShapeEnv()
+        x = torch.randn(123, 4, 16, device="cuda", dtype=torch.bfloat16)
+        offsets = torch.tensor([0, 30, 60, 90, 123], device="cuda")
+        max_seqlen = 40
+        # with FakeTensorMode(shape_env=shape_env) as fake_mode:
```

Comment:
Do you want to remove these commented out code? Or do they both work?

Reply:
oh, thanks for catching that